### DISCUSSION TITLE
How do I donate to your project?

### DISCUSSION THREAD

**Discussion by joshring** [agronholm/typeguard#214] [Q&A]
How do I donate to your project?

**Comment by agronholm** [comment]
I've never been asked that. I don't think I would even want to accept donations since I would then feel bad about not working on the project regularly.

**Comment by joshring** [comment]
It's not a quid pro quo, more of a show of appreciation.

**Comment by dimaqq** [comment]
Random outsider ideas:
* fix a bug, contribute through code, donate your time :)
* advertise the library, help with awareness, get more folk involved :)
* next time you're in the same city or at same conference, take the author out for coffee ☕️

**Comment by Trigve** [comment]
@agronholm It's shame that you don't accept donations because I would gladly donate as I'm using your library in our embedded python for runtime type checking and it helped me catch some nasty bugs I would be for hunting for days otherwise.

**Comment by agronholm** [comment]
This is something I need to check with the police (in my country, collecting donations requires a permit which is only granted to organizations). It is unclear if an individual is allowed to collect donations on an international platform.

**Comment by agronholm** [comment]
I've received some donations via https://thanks.dev/ – you can try with that. I can't officially advertise that, though, as it would run afoul of money collection laws in my country.
